ubuntu-advantage

```diff
@@ -82,6 +34,7 @@ enable_esm() {
 disable_esm() {
     if [ -f "$REPO_LIST" ]; then
         mv "$REPO_LIST" "${REPO_LIST}.save"
+        rm  "$APT_KEYS_DIR/$REPO_KEY_FILE"
```

Comment:
we do print a message when enabling and disabling telling the use to run `apt-get update`